ramls/examples/userSummaries.sample

```diff
@@ -0,0 +1,47 @@
+{
+  "userSummaries": [
+    {
+      "id": "54778f48-9d4d-488d-8bae-a3eaaf5b9d2a",
+      "outstandingFeeFineBalance": 10,
+      "numberOfOpenFeesFinesForLostItems": 1,
+      "numberOfLostItems": 1,
+      "openLoans": [
+        {
+          "loanId": "6545fd94-90a5-4ea4-a426-95f50e599884",
+          "dueDate": "2020-04-29T09:37:39.000+0000",
+          "returnedDate": "2020-04-30T09:37:39.000+0000",
+          "recall": false
+        }
+      ],
+      "openAccounts": [
+        {
+          "accountId": "62f9bd92-c4cc-4523-b434-81ef09cd8bda",
+          "balance": 10,
+          "feeFineId": "fda05c38-d8db-4eab-8d02-abf59544e6ef"
+        }
+      ]
+    },
+    {
+      "id": "fda05c38-d8db-4eab-8d02-abf59544e6ef",
+      "outstandingFeeFineBalance": 20,
```

Comment:
Shouldn't this property have decimal value?

Reply:
updated